api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/WeightMappings.kt

```diff
@@ -211,17 +159,22 @@ internal object WeightMappings {
                 MOVING_MEAN_DATA_PATH_TEMPLATE,
                 MOVING_VARIANCE_DATA_PATH_TEMPLATE
             )
-        val variables = mutableMapOf(
-            Pair(batchNormMovingMeanVarName(layer.name), layerBatchNormPaths.movingMeanPath),
-            Pair(batchNormMovingVarianceVarName(layer.name), layerBatchNormPaths.movingVariancePath)
+        return mapOfNotNull(
+            layer.movingMean to layerBatchNormPaths.movingMeanPath,
+            layer.movingVariance to layerBatchNormPaths.movingVariancePath,
+            layer.gamma to layerBatchNormPaths.gammaPath,
```

Comment:
does it correctly handle the situation when the layer.scale= false or layer.center = false?